src/protocol/boolean/generate_random_bits/semi_honest.rs

```diff
@@ -0,0 +1,54 @@
+use super::{RandomBits, Step};
```

Comment:
This is exactly the same as the old `generate_random_bits()`. No need to review.